client/pages/screens/Login/login.tsx

```diff
@@ -74,50 +52,27 @@ function LoginPage () {
               </div>
               <button
                 type='submit'
-                className='w-4/5 flex justify-center bg-secondary hover:bg-main text-white font-bold py-2 px-4 rounded-3xl mb-6'
+                className='w-full bg-secondary hover:bg-main text-white font-bold py-2 px-4 rounded-3xl mb-6'
                 onClick={navigateToHome}
               >
                 Log in
               </button>
               <button
                 type='submit'
-                className='w-4/5 flex justify-center bg-secondary hover:bg-main text-white font-bold py-2 px-4 rounded-3xl'
+                className='w-full bg-secondary hover:bg-main text-white font-bold py-2 px-4 rounded-3xl'
                 onClick={navigateToHome}
               >
                 Continue with Google
               </button>
             </form>
-          </div>
-        </div>
-      </main>
-
-      <footer className='bg-main text-white p-4 h-40'>
-        <div className='flex justify-between'>
-          <div className='flex items-start'>
-            <div className='mr-2'>
-              <Image src={logo} alt='' className='h-20 w-20 object-cover' />
-            </div>
-            <div className='overflow-wrap'>
-              <span className='italic text-lg font-medium max-w-xs break-words'>
-                About Us:
-              </span>
-              <h2 className='max-w-[40%]'>
-                At the core of FlexiPlanner lies innovation by the Liberal
-                Party, offering a flexible way to organize. Empowering through
-                technology for a brighter, efficient tomorrow.
-              </h2>
-            </div>
-          </div>
-          <div className='flex flex-col justify-start ml-4'>
-            <div className='mr-4'>
-              <span className='italic text-lg font-medium'>Contact Us:</span>
-              <h1 className='text-lg font-medium'>Feel free to reach out:</h1>
-              <p>Email: liberalparty@gmail.com</p>
-              <p>Contact Number: 09696969699</p>
+            <div className='mt-4 text-center'>
+              <button className='text-main hover:underline cursor-pointer'>
+                Don't have an account?
```

Comment:
change Don't to Don&apos;t, d na pwede magamit sang " ' "/apostrophe dapat &apos;